cmd/common.go

```diff
@@ -259,27 +258,15 @@ func verifyCert(caCert []byte) (string, error) {
 	return string(caCert), nil
 }
 
-func loadConfig(ctx *cli.Context) (config.Config, error) {
+func GetConfigPath(ctx *cli.Context) string {
 	// path will always be set by the global flag default
 	path := ctx.GlobalString("config")
-	path = filepath.Join(path, cfgFile)
-
-	cf := config.Config{
-		Path:    path,
-		Servers: make(map[string]*config.ServerConfig),
-	}
-
-	content, err := ioutil.ReadFile(path)
-	if os.IsNotExist(err) {
-		return cf, nil
-	} else if err != nil {
-		return cf, err
-	}
-
-	err = json.Unmarshal(content, &cf)
-	cf.Path = path
+	return filepath.Join(path, cfgFile)
```

Comment:
It might even be a good idea to move `cfgFile` to `config.go` and then `LoadFromPath` would accept a directory instead. But I think it's okay as well like this.

Reply:
Agreed! In fact, a few of these functions in this dumping ground (the `common` package) could be moved to the `config` package. `GetConfigPath`, `loadConfig` are the first among them, and the constant as well. But this refactoring is likely outside the scope of this PR.